modules/base/rendering/renderablemodel.cpp

```diff
@@ -126,7 +132,7 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo BlendingOptionInfo = {
-        "BledingOption",
+        "BlendingOption",
         "Blending Options",
         "Debug option for blending colors."
```

Comment:
Same here.  Would be good to have some more comprehensive description

Reply:
I agree because I'm not sure what this property does :sweat_smile: